en/complete_vehicles/atl_mantis_edu.md

```diff
@@ -0,0 +1,139 @@
+# ATL Mantis EDU
+
+:::warning
+The ATL Mantis EDU is currently work in progress. Use with caution.
+:::
+
+The ATL Mantis EDU is based on the ready-to-fly Yuneec Mantis G and geared towards education.
+
+## Purchase
+
+The ATL Mantis EDU version will be released soon.
+TODO: add purchase link
+
+## Hardware overview
+
+The Mantis broadly features the following hardware components:
+- Autopilot/FMU based on STM32F7.
+- Gimbal controller.
+- Camera system on a chip with WiFi (forwards MAVLink messages from autopilot and gimbal via Wifi)
+- Downward-facing flow module with laser range sensor.
+
+## Update components
+
+The files to update the Mantis can currently be found on [GoogleDrive](https://drive.google.com/drive/u/1/folders/1tDo9AktWtB47LRzaRaMxDkONTNQHsjCG).
```

Comment:
Can we get this on github somewhere. That would allow a direct link and remove browser dependencies (I had one because I have addblocker). 

My first reaction when I was this was "why do I need to update Mantis". Probably that question should be answered in the intro where you have "The ATL Mantis EDU is based on the ready-to-fly Yuneec Mantis G and geared towards education."

Reply:
Note, this ONLY has firmware.bin for me, which looks like the camera file. It might be good to list all of the files here that someone would expect to see, along with their purpose.